### PR TITLE
REMOVES TGUI

### PR DESCRIPTION
**Pull request by stylemistake**
## TGUI is gone, long live TGUI!

crabrave

Completely removed everything related to old tgui (including all hacks to dual boot both UIs), and tgui-next replaces the old one (physically, by moving the folder from `/tgui-next` to `/tgui`).

Preferrably merged after actioninja's **FINAL RECKONING** #49326.

This will obviously cause some issues in downstreams, especially those who still contribute to the legacy tgui, so please discuss.

## Changelog
:cl:
del: Removed tgui
refactor: tgui-next is now the new tgui
code: Updated "interface not found" screen.
/:cl:

### CODE REVIEW

**Comment by XDTM** [comment]
Renaming it tgui makes sense, i suppose, but it's definitely going to cause some confusion when talking about the old version, or when trying to tell which version a server is using.

**Comment by Tlaltecuhtli** [comment]
😎 👌

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -258,11 +258,11 @@ This prevents nesting levels from getting deeper then they need to be.
 	* Areas should not be var-edited on a map to change it's name or attributes. All areas of a single type and it's altered instances are considered the same area within the code, and editing their variables on a map can lead to issues with powernets and event subsystems which are difficult to debug.
 
 ### User Interfaces
-* All new player-facing user interfaces must use TGUI-next; TGUI is deprecated. 
+* All new player-facing user interfaces must use TGUI.
 * Raw HTML is permitted for admin and debug UIs.
-* Documentation for TGUI-next can be found at: 
-	* [tgui-next/README.md](../tgui-next/README.md)
-	* [tgui-next/tutorial-and-examples.md](../tgui-next/docs/tutorial-and-examples.md)
+* Documentation for TGUI-next can be found at:
```
Comment (BadSS13Player) [inline .github/CONTRIBUTING.md]:
```suggestion
* Documentation for TGUI can be found at:
```

**Comment by Ty-the-Smonk** [comment]
“It’s finally over”
“It only took 30 years”

**Comment by Neo-0** [comment]
please keep the name tgui-next in the folder

**Comment by trollbreeder** [comment]
Do the techfabs run on nano-ui or TGUI?

**Comment by optimumtact** [comment]
@actioninja @stylemistake impressive work, kudos to you both

**Comment by Tharcoonvagh** [comment]
exosuit fabricator?
rnd stuff?
huhh

**Comment by spookydonut** [comment]
finally!

**Comment by actioninja** [comment]
@trollbreeder @Tharcoonvagh none of these actually use tgui, they're all raw html.

**Comment by Timonkeyn** [comment]
What about the jail cell timer

**Comment by actioninja** [comment]
that's already been converted.

File: tgui/docs/archives/the-holy-bible-of-tgui-by-arcane.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
+# The Holy Bible of TGUI with Arcane and a shot of Vodka
```
Comment (Cyberboss) [inline tgui/docs/archives/the-holy-bible-of-tgui-by-arcane.md]: This file looks out of date?

Comment (stylemistake) [reply tgui/docs/archives/the-holy-bible-of-tgui-by-arcane.md]: It is, I added it for fun.

Comment (actioninja) [reply tgui/docs/archives/the-holy-bible-of-tgui-by-arcane.md]: Add some kind of clarification that this is old and refers to a deprecated version.

**Comment by actioninja** [comment]
Not sure about the tgui rename, I think it's going to cause some confusion.
If you're confident that this is how you want to do it though, this looks ready to go.

**Comment by actioninja** [comment]
I've decided I don't care that much and people will get used to it.

**Pull request merged**